libraries/mbed/targets/cmsis/TARGET_Freescale/TARGET_KLXX/TARGET_KL05Z/MKL05Z4.h

```diff
@@ -2008,7 +2008,7 @@ typedef struct {
 #define OSC_CR_SC2P_SHIFT                        3
 #define OSC_CR_EREFSTEN_MASK                     0x20u
 #define OSC_CR_EREFSTEN_SHIFT                    5
-#define OSC_CR_ERCLKEN_MASK                      0x80u
+#define OSC_CR_ERCLKEN_MASK                      0x106u
```

Comment:
The mask is 0x80, which is correct.  You have to change the line 149 in the startup file to add capacitors, using MASK for them as SCxP, which are in above this line. Please correct this.